tests/acceptance/features/webUIFiles/fileFolderActionMenu.feature

```diff
@@ -0,0 +1,84 @@
+Feature: User can see the file or folder actions menu options
+  As a user
+  I want to be able to see the actions menu options of file or folder
+  So that the menu options of the selected file type or folder are visible to me
+
+Background: prepare user and files
+    Given user "user1" has been created with default attributes
+    And user "user1" has uploaded file with content "pdf file" to "lorem.pdf"
+    And user "user1" has logged in using the webUI
+    And the user has browsed to the files page
+    
+  Scenario: select folder to see actions menu options
+    When the user picks the row of folder "simple-folder" in the webUI
+    Then the app-sidebar for folder "simple-folder" should be visible on the webUI
+    And the following items should be visible in the actions menu on the webUI
+      | items                   |
+      | open folder             |
+
+  Scenario: select text file to see actions menu options
+    When the user picks the row of file "lorem.txt" in the webUI
+    Then the app-sidebar for file "lorem.txt" should be visible on the webUI
+    And the following items should be visible in the actions menu on the webUI
```

Comment:
How about adding a new step here
```suggestion
    And only the following items should be visible in the actions menu on the webUI
```
with this we wont have to check this 2 times as long as there are not any uncertain menu items